schemas/metaschema/systemDomains.schema

```diff
@@ -0,0 +1,9 @@
+{
+  Type: Enum('string', 'number', 'bigint', 'boolean', 'object', 'function'),
+  Subtype: Enum('double', 'int'),
+  String: { type: 'string' },
+  Logical: { type: 'boolean' },
+  Number: { type: 'number' },
+  Lambda: { type: 'function' },
+  StringsArray: { type: 'object', class: 'Array' }
```

Comment:
Please add a validator, that will check if every element of an array is a string.